src/options-from-args.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import type {Options} from "./types.js";
+import type {minimistOutput} from "minimist";
+
+/**
+ * Convert arguments to options.
+ */
+export const optionsFromArgs = (args: minimistOutput): $Shape<Options> => {
+    const options: $Shape<Options> = {};
```

Comment:
nit: maybe use `Partial` instead of `$Shape`.

Reply:
It doesn't work for this situation.

Reply:
I thought that `Partial` was just a better version of `$Shape`.  Do they handle optionality differently?

Reply:
No, so `Partial` as I understand it, says "I am a type that is any combination of the values within this type" whereas `$Shape` says, "I look like this type, but all my values are optional" or something along those lines.

So `T` can be considered equal to `$Shape<T>` it seems, but `T` is not equal to `Partial<T>`.

At least, that's my basic understanding.